apps/prairielearn/src/sprocs/assessment_instances_insert.sql

```diff
@@ -8,8 +8,7 @@ CREATE FUNCTION
         IN time_limit_min integer DEFAULT NULL,
         IN date timestamptz DEFAULT NULL,
         IN client_fingerprint_id bigint DEFAULT NULL,
-        OUT assessment_instance_id bigint,
-        OUT new_instance_question_ids bigint[]
```

Comment:
AFAICT this column is never used when returned from `assessment_instances_update`; we might be able to remove it from that for a small bit of cleanup? I'm also happy to merge this PR as-is though.

Reply:
I thought of that and even drafted that change, but just in case there is any plan in the near future to e.g., highlight the new instance questions this may become useful. The computation of new instances is used internally in assessment_instances_update, but indeed not outside, so one could convert the parameter into a local variable. I'll leave that for a future PR.